.github/ISSUE_TEMPLATE/1.bug_report.yml

```diff
@@ -32,12 +26,6 @@ body:
       description: 'For example: Chrome, Safari'
     validations:
       required: true
-  - type: input
-    attributes:
-      label: What operating system are you using?
-      description: 'For example: macOS, Windows'
-    validations:
-      required: true
```

Comment:
What about if the user has a version of Next.js prior to `next info` existing?

Reply:
Then instead of removing, we could make these optional?

But then we have to make the `next info` field also optional.

We kind of risk people to not providing the info at all. :thinking:

Reply:
I think we'll need to keep these questions required until `next info` is shipped on stable.